SeedVaultSimulator/src/main/java/com/solanamobile/seedvaultimpl/data/SeedRepository.kt

```diff
@@ -162,23 +162,18 @@ class SeedRepository @Inject constructor(
                 check(!seeds.value.containsKey(nextSeedId)) { "Seed repository already contains an entry for seed $nextSeedId" }
                 newSeedRecordBuilder.seedId = nextSeedId
                 id = nextSeedId
+                val changeNotification = ChangeNotification(
```

Comment:
None of the changes in this file are strictly necessary, but it cuts down on unnecessary change notifications when nothing actually changed, which improves performance of apps that make use of content notifications.